privacyidea/lib/eventhandler/tokenhandler.py

```diff
@@ -391,6 +391,9 @@ def do(self, action, options=None):
                                         "has no email address!".format(user))
                 elif tokentype == "motp":
                     init_param['motppin'] = handler_options.get("motppin")
+                    
+                elif tokentype == "certificate":
+                    init_param['request'] = request.all_data.get("request")
```

Comment:
What is your intention with enrolling a certificate.
Where does actually the "request" parameter come from, which is usually not contained in the originial http request.

The problem of this approach is, that the originial https request needs to know, that there is an event handler configured, that needs an additional parameter. 
We should think about a more generic way to solve this.